static/js/init.js

```diff
@@ -3,6 +3,7 @@
 /* global require, window, document */
 
 var $ = require('jquery');
+var keyboard = require('keyboardjs');
```

Comment:
FWIW, I could see doing more with keyboard shortcuts down the road but
don't feel too strongly about it or whether we should include a library.

On Tue, Jun 9, 2015 at 4:32 PM, Joshua Carp <notifications@github.com>
wrote:

> In static/js/init.js
> <https://github.com/18F/openFEC-web-app/pull/253#discussion_r32076153>:
>
> > @@ -3,6 +3,7 @@
> >  /* global require, window, document */
> >
> >  var $ = require('jquery');
> > +var keyboard = require('keyboardjs');
>
> We don't need it, but copying and pasting key codes from the internet is
> extra work and error-prone. We're also using the library in the typeahead
> module, but not in depth--really, all it does is key mapping lookups and a
> more concise event listener. It's about 10k minified. This would be easy to
> revert if you want--I kind of like it, but no strong preferences.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/openFEC-web-app/pull/253/files#r32076153>.
>



-- 
Noah Manger
18F <http://18f.gsa.gov> | Work: 415-696-6146